db/migrate/20190212160351_update_overrides_column_type.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateOverridesColumnType < ActiveRecord::Migration[5.2]
+  def up
+    rename_column :overrides, :override_reasons, :override_reasons
+    rename_column :allocations, :override_reason, :override_reasons
+    change_column :overrides, :override_reasons, :string, using: "override_reasons::character varying[]"
```

Comment:
Postgres has an array data type - perhaps more appropriate to use this?
See https://blog.arkency.com/2014/10/how-to-start-using-arrays-in-rails-with-postgresql/

Reply:
I initially tried this, but as we're changing a column that already exists rather than creating a new one the migration failed as was unable to cast automatically.  Therefore we need to use this syntax; which is very similar to the issue we faced in this migration: `update_columns_in_po_ms.rb` where we were trying to change the data type.